OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -259,6 +259,8 @@ public interface IActorMap
 
 		WDist LargestActorRadius { get; }
 		WDist LargestBlockingActorRadius { get; }
+
+		event Action<CPos> CellUpdated;
```

Comment:
Can you change the commit message (currently "Added UpdateCell event to ActorMap") to use "CellUpdated"?